RELEASE_NOTES.md

```diff
@@ -1,5 +1,152 @@
+- Fixed a bug where the `podman run` and `podman create` commands would deadlock when trying to create a container that mounted the same named volume at multiple locations (e.g. `podman run -v testvol:/test1 -v testvol:/test2`) ([#8221](https://github.com/containers/podman/issues/8221)).
+- Fixed a bug where the parsing of the `--net` flag to `podman build` was incorrect ([#8322](https://github.com/containers/podman/issues/8322)).
+- Fixed a bug where the `podman build` command would print the ID of the built image twice when using remote Podman ([#8332](https://github.com/containers/podman/issues/8332)).
+- Fixed a bug where the `podman stats` command did not show memory limits for containers ([#8265](https://github.com/containers/podman/issues/8265)).
+
+### API
+- The Compat Create endpoint for Container has received a major refactor to share more code with the Libpod Create endpoint, and should be significantly more stable.
+- A Compat endpoint for exporting multiple images at once, `GET /images/get`, has been added ([#7950](https://github.com/containers/podman/issues/7950)).
+- The Compat Network Connect and Network Disconnect endpoints have been added.
+- Endpoints that deal with image registries now support a `X-Registry-Config` header to specify registry authentication configuration.
+- The Compat Create endpoint for images now properly supports specifying images by digest.
+- The Libpod Build endpoint for images now supports an `httpproxy` query parameter which, if set to true, will forward the server's HTTP proxy settings into the build container for `RUN` instructions.
+- The Libpod Untag endpoint for images will now remove all tags for the given image if no repository and tag are specified for removal.
+- Fixed a bug where the Compat Logs endpoint for containers did not send a newline character after each log line.
+- Fixed a bug where the Compat Logs endpoint for containers would mangle line endings to change newline characters to add a preceding carriage return ([#7942](https://github.com/containers/podman/issues/7942)).
+- Fixed a bug where the Compat Inspect endpoint for Containers did not properly list the container's stop signal ([#7917](https://github.com/containers/podman/issues/7917)).
+- Fixed a bug where the Compat Inspect endpoint for Containers formatted the container's create time incorrectly ([#7860](https://github.com/containers/podman/issues/7860)).
+- Fixed a bug where the Compat Inspect endpoint for Containers did not include complete network information on the container.
+- Fixed a bug where the server could panic if a client closed a connection midway through an image pull ([#7896](https://github.com/containers/podman/issues/7896)).
+- Fixed a bug where the Compat Create endpoint for volumes returned an error when a volume with the same name already existed, instead of a 201 ([#7740](https://github.com/containers/podman/issues/7740)).
```

Comment:
```suggestion
- Fixed a bug where the Compat Create endpoint for volumes returned an error when a volume with the same name already existed, instead of a 201 error ([#7740](https://github.com/containers/podman/issues/7740)).
```